packages/imperative/src/plugins/cmd/validate/validate.definition.ts

```diff
@@ -35,6 +35,24 @@ export const validateDefinition: ICommandDefinition = {
       required: false
     }
   ],
+  options: [
+    {
+      name: "fail-on-error",
+      aliases: ["foe"],
+      type: "boolean",
+      description: "Enables throwing an error and setting an error code if plugin validation detects an error",
+      required: false,
+      defaultValue: false
+    },
+    {
+      name: "fail-on-warning",
+      aliases: ["fow"],
+      type: "boolean",
+      description: "Treat validation warnings as errors. Requires fail-on-error.",
+      required: false,
```

Comment:
Should this have defaultValue of false? I guess that's the default behavior so not necessarily required. Just curious because fail-on-error has defaultValue of false defined 🙂

Reply:
I did not define it because of the implicit requirement of the other value. I can remove the default for the other if you like, I just added it because we are planning to make it true on next.

Reply:
I'm in favor of consistency 😅 
either add it for both, or remove it for both 😋